src/main/java/com/spotinst/sdkjava/model/bl/ocean/ecs/ClusterInstanceTypes.java

```diff
@@ -0,0 +1,80 @@
+package com.spotinst.sdkjava.model.bl.ocean.ecs;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class ClusterInstanceTypes {
```

Comment:
Didn't find this parameter at cluster level.

Reply:
Parameters keyPair and Instance Types are available in JSON when we check 'view configuration' for Cluster on UI. So Added it. 
So As Discussed, Currently aligning with postman and removing those parameters.